charts/sysdig-deploy/README.md

```diff
@@ -203,6 +205,29 @@ The following table lists the configurable parameters of the sysdig-deploy chart
 | `rapidResponse`                         | Config specific to [Sysdig Rapid Response](#rapid-response)                                                             | `{}`      |
 | `rapidResponse.enabled`                 | Enable Rapid Response component in this chart                                                                           | `""`      |
 
+## Sysdig Plan Settings
```

Comment:
The onboarding page in the UI shows this under "helm"

```
kubectl create ns sysdig-agent
helm repo add sysdig https://charts.sysdig.com
helm repo update
helm install sysdig-agent --namespace sysdig-agent \
    --set global.sysdig.accessKey=e64a1780-5bbd-4083-8ce1-418c2bb0a356 \
    --set global.sysdig.region=custom \
    --set nodeAnalyzer.secure.vulnerabilityManagement.newEngineOnly=true \
    --set global.kspm.deploy=true \
    --set nodeAnalyzer.nodeAnalyzer.benchmarkRunner.deploy=false \
    --set global.clusterConfig.name=<CLUSTER_NAME> \
    sysdig/sysdig-deploy 
```
How will that change with the introduction of the new global plan settings? For example, are the nodeAnalyzer settings still required?

Reply:
With these changes as they are the instructions could become slightly more complicated as secure customers can now optionally set `global.sysdig.secure=true` in the `helm install ...` command. The reason for this is because the other settings, particularly `nodeAnalyzer.secure.vulerabilityManagement.newEngineOnly=true` and `nodeAnalyzer.nodeAnalyzer.benchmarkRunner.deploy=false` are manipulating how the subcharts work.

One change I can do is include the the KSPM collector as a part of `global.sysdig.secure=true`. There was actually a whole discussion on this topic yesterday (disabling bencmark v2 and enabling kspm-collector when `global.sysdig.secure` is set). I was planning on doing that change in a follow up PR, but it wouldn't be too hard to put into this one. @balinaveen , what are your thoughts?

Reply:
>One change I can do is include the the KSPM collector as a part of global.sysdig.secure=true. There was actually a whole discussion on this topic yesterday (disabling bencmark v2 and enabling kspm-collector when global.sysdig.secure is set). I was planning on doing that change in a follow up PR, but it wouldn't be too hard to put into this one. @balinaveen , what are your thoughts?
On second thought, I would not like to do this as a part of this PR. I would like to engage the KSPM folks for this change so that they are aware.

Reply:
Yeah, let's do the ksmp changes later. If the onboarding UI as it exists today will still work, then that's OK. I was hoping things would be simpler, but that can wait. Let's postpone the update to the UI until all changes needed to simplify things are in the helm chart.